File: src/domain/activity/project/exception/project-not-found.exception.ts
import { HttpStatus } from '@nestjs/common';

import { ApiException } from '@wink/swagger';

export class ProjectNotFoundException extends ApiException {
  constructor() {
    super({
      swagger: '프로젝트를 찾을 수 없음',
      message: '프로젝트를 찾을 수 없습니다.',
      code: HttpStatus.NOT_FOUND,
    });
  }
}

File: src/main/java/com/github/kmu_wink/wink_official/domain/program/project/exception/ProjectNotFoundException.java
package com.github.kmu_wink.wink_official.domain.program.project.exception;

import org.springframework.http.HttpStatus;

import com.github.kmu_wink.wink_official.common.api.exception.ApiException;

public class ProjectNotFoundException extends ApiException {

	public ProjectNotFoundException() {
		super(HttpStatus.NOT_FOUND, "프로젝트를 찾을 수 없습니다.");
	}
}
